en/restore-from-azblob-using-br.md

```diff
@@ -46,106 +64,165 @@ Before restoring backup data on Azure Blob Storage to TiDB using BR, take the fo
+    kubectl apply -f restore-point-azblob.yaml
     ```
 
-    The content of `restore-azblob.yaml` is as follows:
+    The content of `restore-point-azblob.yaml` is as follows:
 
-     ```yaml
+    ```yaml
     ---
     apiVersion: pingcap.com/v1alpha1
     kind: Restore
     metadata:
-      name: demo2-restore-azblob
-      namespace: test2
+    name: demo3-restore-azblob
+    namespace: restore-test
     spec:
-      serviceAccount: tidb-backup-manager
+      restoreMode: pitr
       br:
-        cluster: demo2
-        sendCredToTikv: false
-        clusterNamespace: test2
-        # logLevel: info
-        # statusAddr: ${status_addr}
-        # concurrency: 4
-        # rateLimit: 0
-        # timeAgo: ${time}
-        # checksum: true
-      # Only needed for TiDB Operator < v1.1.10 or TiDB < v4.0.8
-      to:
-        host: ${tidb_host}
-        port: ${tidb_port}
-        user: ${tidb_user}
-        secretName: restore-demo2-tidb-secret
+        cluster: demo3
+        clusterNamespace: test3
       azblob:
-        secretName: azblob-secret-ad
+        secretName: azblob-secret
         container: my-container
-        prefix: my-folder
+        prefix: my-log-backup-folder-pitr
+      pitrRestoredTs: "2022-10-10T17:21:00+08:00"
+      pitrFullBackupStorageProvider:
+        azblob:
+          secretName: azblob-secret
+          container: my-container
+          prefix: my-full-backup-folder-pitr
     ```
 
-When configuring `restore-azblob.yaml`, note the following:
+    When you configure `backup-azblob.yaml`, note the following:
 
-- For more information about Azure Blob Storage configuration, refer to [Azure Blob Storage fields](backup-restore-cr.md#azure-blob-storage-fields).
-- Some parameters in `.spec.br` are optional, such as `logLevel`, `statusAddr`, `concurrency`, `rateLimit`, `checksum`, `timeAgo`, and `sendCredToTikv`. For more information about BR configuration, refer to [BR fields](backup-restore-cr.md#br-fields).
-- For v4.0.8 or a later version, BR can automatically adjust `tikv_gc_life_time`. You do not need to configure `spec.to` fields in the `Restore` CR.
-- For more information about the `Restore` CR fields, refer to [Restore CR fields](backup-restore-cr.md#restore-cr-fields).
+    - `spec.restoreMode`: when you perform PITR, set this field to `pitr`. The default value of this field is `snapshot`, which means snapshot backup.
 
-After creating the `Restore` CR, execute the following command to check the restore status:
+2. Wait for the restoration operation to complete:
+
+    ```shell
+    kubectl get jobs -n restore-test
+    ```
 
-{{< copyable "shell-regular" >}}
+    ```
+    NAME                           COMPLETIONS   ...
+    restore-demo3-restore-azblob   1/1           ...
+    ```
 
-```shell
-kubectl get rt -n test2 -o wide
-```
+    You can also check the restoration status by using the following command:
+
+    ```shell
+    kubectl get rt -n restore-test -o wide
+    ```
 
```

Comment:
update the explaination about `restoreMode`
https://github.com/pingcap/docs-tidb-operator/pull/2046/commits/e28d604e2a32dc755b5c54c05f7cae21adfa05e9

Reply:
https://github.com/pingcap/docs-tidb-operator/commit/e28d604e2a32dc755b5c54c05f7cae21adfa05e9#diff-83155639f2ab0ae671231426ac5676d82715077e19f4d756560be06cd8fdbf51R220

Reply:
I moved `restoreMode` in both Chinese and English here: https://github.com/pingcap/docs-tidb-operator/pull/2048/files#diff-8ba69f032198f88a0d3d7dd9c4dd519fcaaeaf03495e67f3f54253df6b5ac5a1R205-R207 This is where the users are mostly likely to notice `restoreMode`

Reply:
get